### DISCUSSION TITLE
Welcome to tibia.py Discussions!

### DISCUSSION THREAD

**Discussion by Galarzaa90** [Galarzaa90/tibia.py#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]


**Reply by Galarzaa90** [reply]
Are you using the client or just the parser?

**Reply by ghost** [reply]
I do use just a parser, thanks for a hint! Fetch_auction method from Client indeed has what I need.
